crates/testing/src/overall_safety_task.rs

```diff
@@ -225,6 +231,18 @@ impl<TYPES: NodeType, I: TestableNodeImplementation<TYPES>> TestTaskState
                 failed_views: self.ctx.failed_views.clone(),
             }));
         }
+
+        // We should really be able to include a check like this:
```

Comment:
Should we just remove this and instead file an issue with this context? Just to avoid it getting lost?

Reply:
I really like having a comment in the code, but I've also created an issue: https://github.com/EspressoSystems/HotShot/issues/3334

Reply:
Yeah that make sense to me!